portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -10,9 +10,11 @@
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
+   // private String requestSenderName = " ";
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    //response.getWriter().println("Hello" + requestSenderName);
```

Comment:
Seems like you were looking to print out something unique about the request, which is a good thought! I don't see anything that specifies the name of the sender directly, but there's a few  methods you could try out if you wanted to print something unique based off the request, such as calling `request.getHeader()` or `request.getRequestURL()`. 

I got those methods from the info about the [HttpServletRequest documentation](https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletRequest.html). Definitely not required, but thought I'd share to build off what you were thinking about adding!

Reply:
Thank you Gina! The resources is helpful!